src/pages/get-started/api-security.md

```diff
@@ -7,6 +7,240 @@ This topic describes best practices for [API security](https://owasp.org/www-pro
+--backpressure-logger-redis-password=BACKPRESSURE-LOGGER-REDIS-PASSWORD                    Redis server password
+--backpressure-logger-redis-user=BACKPRESSURE-LOGGER-REDIS-USER                            Redis server user
+--backpressure-logger-id-prefix=BACKPRESSURE-LOGGER-ID-PREFIX                              ID prefix for keys
+```
+
+<InlineAlert variant="info" slots="text"/>
+
+The data (the request time and the identifier) is temporarily stored in Redis. Registered users are identified by their user ID. Non-registered users are identified by their external IP address.
+
+The following command will add a new connection to the Redis server:
+
+Redis server:
+- Host: 195.34.23.5
+- Port: 9345
+- Password: s0M3StR0NgP@SsW0Rd
+- User: SomeUser
+
+```bash
+$ bin/magento setup:config:set \
+    --backpressure-logger=redis \
+    --backpressure-logger-redis-server=195.34.23.5 \
+    --backpressure-logger-redis-port=9345 \
+    --backpressure-logger-redis-timeout=1 \
+    --backpressure-logger-redis-persistent=persistent \
+    --backpressure-logger-redis-db=3 \
+    --backpressure-logger-redis-password=s0M3StR0NgP@SsW0Rd\
+    --backpressure-logger-redis-user=SomeUser \
+    --backpressure-logger-id-prefix=some_pref
+```
+
+After the command is executed, the following configuration is added to the `app/etc/env.php` file.
+
+```php
+[
+//...
+    'backpressure' => [
+        'logger' => [
+            'type' => 'redis',
+            'options' => [
+                'server' => '195.34.23.5',
+                'port' => 9345,
+                'timeout' => 1,
+                'persistent' => 'persistent',
+                'db' => '3',
+                'password' => 's0meStr0ngPassw0rd',
+                'user' => 'SomeUser'
+            ],
+            'id-prefix' => 'some_pref'
+        ]
+    ]
+//...
+];
+```
+
+Use the following commands to enable rate limiting:
+
+1. Enable (`1`) or disable (`0`) rate limiting for placing orders:
+
+    ```bash
+    $ bin/magento config:set sales/backpressure/enabled 1
+    ```
+   
+1. Set the request limit per guest (IP address):
+
+    ```bash
+    $ bin/magento config:set sales/backpressure/guest_limit 100
+    ```
+   
+1. Set the request limit for authenticated customers:
+
+    ```bash
+    $ bin/magento config:set sales/backpressure/limit 10
+    ```
+   
+1. Set the amount of time (in seconds) a customer must wait after reaching the `limit`. Supported values `60`, `3600`, `86400` seconds:
+
+    ```bash
+    $ bin/magento config:set sales/backpressure/period 3600
+    ```
+   
+If you need to check a configuration, use the following CLI command:
+
+```bash 
+$ bin/magento config:show | grep backpressure
+```
+
+Example:
+```bash
+$ bin/magento config:show | grep backpressure
+```
+
+```terminal
+sales/backpressure/limit - 10
+sales/backpressure/guest_limit - 100
+sales/backpressure/period - 3600
+sales/backpressure/enabled - 1
+```
+
```

Comment:
I think this should probably be a new section.

Reply:
@jhadobe 
Sorry, more details please, because I can't figure out what part you have in mind

Reply:
@arhiopterecs after reviewing it, i don't think the change i suggested is necessary.